src/fonts.js

```diff
@@ -55,37 +55,51 @@ function funLocaleSpecificFonts () {
 			'ja': {
 				regular: 'LG Display_JP',
 				unicodeRanges:
-					'U+0000-U+00FF, ' +
-					'U+2E80-U+2EFF,' +
-					'U+2F00-U+2FDF,' +
-					'U+3000-U+303F,' +
-					'U+3040-U+309F,' +
-					'U+30A0-U+30FF,' +
-					'U+3200-U+33FF,' +
-					'U+3400-U+4DBF,' +
-					'U+4E00-U+9FFF,' +
-					'U+E000-U+FAFF,' +
-					'U+FF00-U+FFEF'
+					'U+0-FF,' +
+					'U+2E80-2EFF,' +
```

Comment:
Hi @Djspaceg, just curious, is it same behavior with excluding 'U+' from the end range?